### ISSUE TITLE
input var doesn't want to be okay

### ISSUE THREAD

**Issue by fbm-static** [WordPress-Coding-Standards/WordPress-Coding-Standards#384]
The line is:

```
<?php if ( isset ( $_GET['action'] ) && absint( $_GET['action'] ) === 'new' ) : // input var okay ?>
```

Detected access of super global var $_GET, probably need manual inspection
Detected usage of a non sanitized input variable: $_GET

Help!

**Comment by fbm-static** [comment]
I've followed this wiki for above:

https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/wiki/Flagged-superglobal-usage-in-WordPress-VIP

before making this issue.

**Comment by westonruter** [comment]
The sniff is probably not looking for the inline comment after the opening of the control structure block.

As a quick workaround, you can do:

```php
<?php
$is_new = ( isset ( $_GET['action'] ) && absint( $_GET['action'] ) === 'new' ); // input var okay
if ( $is_new ) {
...
```

I think it is more readable too.

**Comment by fbm-static** [comment]
Hello,

Thank you for making a suggestion. However, it still results an error

```
Detected usage of a non sanitized input variable: $_GET
```

Huhu.

**Comment by westonruter** [comment]
Try checking out the latest from the `develop` branch. That should be addressed by #376. If not, you can add `// sanitization okay` as well:

```
$is_new = ( isset ( $_GET['action'] ) && absint( $_GET['action'] ) === 'new' ); // input var okay; sanitization okay
```
